Uni_life_tracker/app/src/main/res/mipmap-anydpi-v26/add_expense.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
instead of these files, you can create vector assets! that way you don't need all of these & can refer to the same icons throughout the project

Reply:
Its not letting me add them so I think I'll just stay consistent and keep those files, I can delete the unused ones

Reply:
Will do!